Source/Engine/Level/Tags.cpp

```diff
@@ -55,6 +55,23 @@ Tag Tags::Get(const StringView& tagName)
     return tag;
 }
 
+Array<Tag> Tags::GetSubTags(Tag parentTag)
+{
+    Array<Tag> subTags;
+    auto _parentTagName = parentTag.ToString();
+
+    for (int i = 0; i < Tags::List.Count(); i++)
+    {
+        auto& subTagName = Tags::List[i];
+        if (subTagName.Contains(_parentTagName) && subTagName != _parentTagName)
```

Comment:
It should be `subTagName.StartsWith()` as tag can contain other tags inside and not be a parent. Also, instead of comparing strings you can compare current tag index (`i+1`) with `parentTag.Index` for faster perf.